src/ape/api/query.py

```diff
@@ -66,10 +71,31 @@ class BlockQuery(_BaseBlockQuery):
 
     @classmethod
     def all_fields(cls) -> List[str]:
+        from .providers import BlockAPI
```

Comment:
Would another way to resolve the circular import be to make a new module `providers/blocks.py` ?
That way, we are not doing anything with the query manager in `providers.py`. I am used to thinking of the provider as the last stop before RPC so I was thrown off by the import of query manager. But it makes sense for the `BlockAPI` so that is why I am thinking it deserves its own module, especially if it also resolves this circular import

Reply:
I'd be for this